source/renderer/app/components/wallet/summary/WalletSummaryAsset.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useState } from 'react';
```

Comment:
good question, I have no idea 👍 maybe there was another feature PR merged into this webpack 5 branch because it required webpack 5? That's kinda the only reason i can think of right now

Reply:
Those files have been removed from this PR as they have been moved to other locations on develop and thus would just create duplication.